index.rst

```diff
@@ -58,7 +58,11 @@ part of which is providing a Python kernel for Jupyter.
 Announcements
 =============
 
-- **IPython 7.3**: several bugfixes and minor improvements (February 18, 2019)
+- **IPython 7.5**: fixes to issues brought by 7.4, documentation changes and other minor updates (April 25, 2019)
+
+- **IPython 7.4**: improvements to the completion system and miscelanious fixes (March 21, 2019)
```

Comment:
miscellaneous*

Reply:
Will do, thanks !

I was confused by:
![DF214EC0-38CE-49DD-A216-058DCD581745](https://user-images.githubusercontent.com/10530980/57568384-0892f880-73e7-11e9-824d-a36b497f49aa.jpeg)
It should also be updated

Reply:
Ha!  That reminds of a an example webpack config that had `minetype` that was so widely copied that tools would accept `minetype` instead of `mimetype`  :smile: